src/sdk/bucketspace_fs.js

```diff
@@ -709,7 +710,7 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
     // so they can be re-used
     async has_bucket_action_permission(bucket, account, action, bucket_path = "") {
         const account_identifier = account.name.unwrap();
```

Comment:
looks like it's not really used and we can use bucket.name.unwrap() the same as line 746

Reply:
removed